File: extractor-core/src/main/java/ru/mydesignstudio/database/metadata/extractor/extractors/model/JobModel.java
package ru.mydesignstudio.database.metadata.extractor.extractors.model;

import lombok.Data;

@Data
public class JobModel {
  private String schemaName;
  private String jobName;
  private String jobStyle;
  private String jobType;
  private String jobAction;
  private String startDate;
  private String schedule;
  private String lastStartDate;
  private String nextRunDate;
  private String state;
}

File: extractor-core/src/main/kotlin/ru/mydesignstudio/database/metadata/extractor/extract/result/JobModel.kt
package ru.mydesignstudio.database.metadata.extractor.extract.result

data class JobModel(
        var schemaName: String,
        var jobName: String,
        var jobStyle: String,
        var jobType: String,
        var jobAction: String,
        var startDate: String,
        var schedule: String,
        var lastStartDate: String,
        var nextRunDate: String,
        var state: String
)